### PR TITLE
fix: Wording "Gift offered" to "Gift given" (#4673)

### PR DESCRIPTION
**Pull request by mgrhm**
Fixes the wording issue raised in issue #4673.

### CODE REVIEW

**Review by asbiin — CHANGES_REQUESTED** [review]
Thank you for your submission.
Unfortunately, you made too much changes.
The only locale file to modify is the `resources/lang/en/people.php` - then regenerate the assets like you did. The other locale file will be updated after localization process.

**Comment by mgrhm** [comment]
I've updated just the `/en/` and `/en-GB/` `people.php` files now and submitted a new commit.

File: resources/lang/en-GB/people.php [hunk 1]
```diff
@@ -308,7 +308,7 @@
     'gifts_delete_cta' => 'Delete',
     'gifts_add_title' => 'Gift management for :name',
     'gifts_add_gift_idea' => 'Gift idea',
-    'gifts_add_gift_already_offered' => 'Gift offered',
+    'gifts_add_gift_already_offered' => 'Gift given',
```
Comment (asbiin) [inline resources/lang/en-GB/people.php:311]: Sorry but `en-GB` is also a translation, and must not be edited (see the header of the file).

**Review by asbiin — CHANGES_REQUESTED** [review]

File: resources/lang/en-GB/people.php (hunk 1 above)
Comment (mgrhm) [reply resources/lang/en-GB/people.php]: Okay. I’ve made a commit that _only_ changes the `en` file, and pushed it.

**Pull request closed without merging**